test/fixtures/fastboot-app/server/middleware/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "fastboot-app-test-middlewares",
+  "version": "1.0.0",
+  "description": "middlewares for fastboot-app",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
+  "license": "MIT",
+  "dependencies": {
```

Comment:
specifically we want to make sure that these dependencies install without error.